torch/_inductor/cpp_builder.py

```diff
@@ -1221,6 +1221,7 @@ def get_cpp_torch_device_options(
 
     if device_type == "xpu":
         definations.append(" USE_XPU")
+        cflags += ["Wno-comment"]
```

Comment:
LGTM. By the way, it is better to add some comments and submit a github Issue to the Intel/LLVM community.

Reply:
Thanks, I'll report this issue.